funnel/templates/notification_feed.html.jinja2

```diff
@@ -17,8 +17,8 @@
           <div class="alert alert--info">
             <p class="alert__text">
               <a href="{{ url_for('add_phone') }}">
-                {{ faicon(icon='exclamation-circle', css_class='fa-icon--margin') }}
-                {%- trans %}To receive timely notifications by SMS, add a phone number{% endtrans %}
+                {{ faicon(icon='exclamation-circle') }}
+                {% trans %}To receive timely notifications by SMS, add a phone number{% endtrans %}
```

Comment:
This change was introduced in a PR yesterday. You are attempting to undo it. Remove this change here.